core/src/main/res/values/strings.xml

```diff
@@ -330,29 +330,29 @@
 
     <!-- Empty list labels -->
     <string name="no_items_header_label">No queued episodes</string>
-    <string name="no_items_label">Add an episode by downloading it, or long press an episode and select \"Add to queue\".</string>
+    <string name="no_items_label">Add episodes by downloading them, or long-press one and select \"Enqueue\".</string>
     <string name="no_shownotes_label">This episode has no shownotes.</string>
     <string name="no_comp_downloads_head_label">No downloaded episodes</string>
-    <string name="no_comp_downloads_label">You can download episodes on the podcast details screen.</string>
-    <string name="no_log_downloads_head_label">No download log</string>
-    <string name="no_log_downloads_label">Download logs will appear here when available.</string>
+    <string name="no_comp_downloads_label">Download episodes from the \'Podcast details\' screen.</string>
+    <string name="no_log_downloads_head_label">No downloads yet</string>
+    <string name="no_log_downloads_label">A history of downloads is displayed here.</string>
     <string name="no_history_head_label">No History</string>
-    <string name="no_history_label">After you listen to an episode, it will appear here.</string>
+    <string name="no_history_label">heard episodes are displayed here.</string>
     <string name="no_all_episodes_head_label">No Episodes</string>
-    <string name="no_all_episodes_label">When you add a podcast, the episodes will be shown here.</string>
+    <string name="no_all_episodes_label">Added podcasts are displayed here.</string>
     <string name="no_new_episodes_head_label">No new episodes</string>
-    <string name="no_new_episodes_label">When new episodes arrive, they will be shown here.</string>
-    <string name="no_fav_episodes_head_label">No favorite episodes</string>
-    <string name="no_fav_episodes_label">You can add episodes to the favorites by long-pressing them.</string>
-    <string name="no_subscriptions_head_label">No subscriptions</string>
-    <string name="no_subscriptions_label">To subscribe to a podcast, press the plus icon below.</string>
+    <string name="no_new_episodes_label">Newly arrived episodes are displayed here.</string>
```

Comment:
Went with "displayed" consistently because "shown" can be misinterpreted here.

Reply:
How could 'shown' be misinterpreted?

Reply:
That playback occurs. Easy to get that wrong in translation.

Reply:
Yes they are not shown, and that is the confusion to avoid.
In Weblate admins can go to a given string and add screenshots, tags, flags and text for it from the UI.